packages/hawtio/src/plugins/shared/util/helpers.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Will format a property to a standart human readable string with its spaces.
+ * It will respect MBean and leave it together
+ * @param str The property to transform
+ * @returns The property with its proper spaces
+ */
+export const tidyLabels = (str: string) =>
+  str
+    .replace(/([a-z])([A-Z])/g, '$1 $2')
+    .replace(/\b([A-Z]+)([A-Z])([a-z])/, '$1 $2$3')
+    .replace('M Bean', 'MBean')
+    .replace(/^./, function (str) {
+      return str.toUpperCase()
+    })
```

Comment:
this impl doesn't seem to cover every case that were covered in the old baseHelpers humanizeValue function. what do you do with the dash-separated cases like `this-is-an-example`?

Reply:
Added with the corresponding test